TrafficCapture/dockerSolution/src/main/docker/migrationConsole/workflows/templates/fullMigration.yaml

```diff
@@ -0,0 +1,322 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: full-migration
+spec:
+  serviceAccountName: argo-workflow-executor
+#  ttlStrategy:
+#    secondsAfterCompletion: 86400  # Delete workflow so many seconds after completion
+  entrypoint: main
+  parallelism: 100
+
+  arguments:
+    parameters:
+      - name: etcd-endpoints
+        value: "http://etcd.ma.svc.cluster.local:2379"
+      - name: etcd-user
+        value: "root"
+      - name: etcd-password
+        value: "password"
+      - name: etcd-image
+        value: "migrations/migration_console:latest"
+      - name: demo-max-sleep-time
+        value: "60"
+      - name: s3-snapshot-configmap
+        value: "s3-snapshot-config"
+
+  templates:
+    # Main workflow template
+    - name: main
+      inputs:
+        parameters:
+          - name: source-migration-configs
+#                {\"source\":{\"endpoint\":\"https://sourceA.example.com\",\"allow_insecure\":true,\"basic_auth\":{\"username\":\"admin\",\"password\":\"admin\"}},\"snapshot-and-migration-configs\":[{\"indices\":[\"index_1\",\"index_2\"],\"migrations\":[{\"metadata\":{\"mappings\":{\"properties\":{}},\"settings\":{}},\"documentBackfillConfigs\":[{\"indices\":[\"index_1\"]},{\"indices\":[\"index_2\"]}]}]}],\"replayer-config\":{\"batchSize\":1000,\"concurrency\":4}},
```

Comment:
nit: Seeing commented out sources in a few places in these templates

Reply:
deleted those